src/services/user.ts

```diff
@@ -4,19 +4,19 @@ export function buildName(firstName: string, lastName?: string) {
 	return lastName ? `${firstName} ${lastName}` : firstName
 }
 
-async function createPlayer(args: {
+async function createUser(args: {
 	db: Database
 	userId: number
 	name: string
 }): Promise<User> {
-	const playerObject = {
+	const userObject = {
 		userId: args.userId,
 		name: args.name
 	} as User
 
-	await args.db.user.insertOne(playerObject)
+	await args.db.user.insertOne(userObject)
```

Comment:
probably `*Object` part is extra

analogy: then we create an array of users we don't name it `usersArray` we name `users`

here the same thing: do not add type into name (it isn't context in this case)

Reply:
@s373r `Object` here explicitly highlights that it's a plain object - not a class/db entry